spec/phenix_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'active_record'
 require 'tmpdir'
 
-SingleCov.covered!
+SingleCov.covered! uncovered: 1
```

Comment:
comment on why it cannot be covered ?

Reply:
SingleCov tells me
```
lib/phenix.rb new uncovered lines introduced (1 current vs 0 configured)
Lines missing coverage:
lib/phenix.rb:77:7-79:10
```
Those are the lines 
```rb
if config.key?(c)
  command << "--#{m}=#{config.fetch(c)}"
end
```
It looks more like a bug in SingleCov actually.

Reply:
not covering the else case

Reply:
There is no else case

Reply:
Ah, I get it.

Reply:
Moving Ruby 2.5 out of this PR.